cmd/bitxhub/start.go

```diff
@@ -156,22 +156,22 @@ func runtimePProf(mode string, id uint64, duration time.Duration) {
 
 	var cpuFile *os.File
 	if mode == "cpu" {
-		cpuPath := fmt.Sprint(rootPath,"cpu-", time.Now().Format("20060102-15:04:05"))
+		cpuPath := fmt.Sprint(rootPath, "cpu-", time.Now().Format("20060102-15:04:05"))
```

Comment:
please change to filepath.join
subPath := fmt.Sprint("cpu-", time.Now().Format("20060102-15:04:05"))
cpuPath := filepath.Join(rootPath,subPath)